wgengine/magicsock/magicsock.go

```diff
@@ -1064,7 +1064,7 @@ func (c *Conn) findAddrSet(addr *net.UDPAddr) *AddrSet {
 	if !ok {
 		return nil
 	}
-	ipp := netaddr.IPPort{ip, uint16(addr.Port)}
+	ipp := netaddr.IPPort{IP: ip, Port: uint16(addr.Port)}
```

Comment:
Likewise, the name of this was already IPPort and there was an ip and port. With this change we have "ip" and "port" 3 times each and (or four if you count the `ipp` variable).

If ip and port were the same type (say, both strings) then I'd absolutely agree they should be named. But in this case the type name says the fields and the design of the netaddr package is such that no new fields are being added.